src/components/toolbar/button.js

```diff
@@ -22,6 +22,8 @@ class Button extends Component {
     } = this.props
     const button = (
       <button
+        role="button"
+        aria-label={this.props.ariaLabel || title}
```

Comment:
simplify: `aria-label={ariaLabel || title}`

Reply:
No problem, I was thinking of adding ariaLabel in the object destructuring, but thinking from a performance optimisation point of view, I feel like it adds a run-time step which isn't necessary. However I do accept that it is more readable so I will make the change to align with your codebase.